TripPlanner - Project/TripPlannerFrontend/src/components/Branding/Logo.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export default function Logo() {
+  return <h3><span style={{ color: '#F53314' }}>Trip</span><img alt="Atomic layout" src="./Vector.png" /><span style={{ color: '#9C1B07' }} >Planner</span></h3>;
```

Comment:
`alt` should have relevant description of the image. Change it to `TripPlanner`, it's more accurate than `Atomic Layout`

Reply:
And looking from more holistic perspective: please use logo from Figma. Just export the whole logo, import it here and make sure it's scaled as in the designs.

Here's the guide to exports in Figma: https://help.figma.com/hc/en-us/articles/360040028114-Guide-to-exports-in-Figma#h_41b394ed-f969-4840-9e6e-c7c2790d5bc5